### PR TITLE
Initializer for worker pool

### PR DESCRIPTION
**Pull request by mosquito**
Add initializer function for WorkerPool workers

### CODE REVIEW

File: aiomisc/worker_pool/pool.py [hunk 1]
```diff
@@ -176,6 +188,21 @@ async def handler() -> None:
...
+                    self.initializer,
+                    self.initializer_args,
+                    dict(self.initializer_kwargs),
+                    initializer_done
+                )
+
+                try:
+                    await initializer_done
+                except Exception:
+                    log.warning("Initializer function has been failed")
```
Comment (dizballanze) [inline aiomisc/worker_pool/pool.py:204]: Maybe re-raise exception here?

File: tests/test_worker_pool.py [hunk 2]
```diff
@@ -84,3 +84,41 @@ async def test_exit_respawn(worker_pool):
...
+
+
+def initializer(*args, **kwargs):
+    global INITIALIZER_ARGS, INITIALIZER_KWARGS
+    INITIALIZER_ARGS = args
+    INITIALIZER_KWARGS = kwargs
+
+
+def get_initializer_args():
+    return INITIALIZER_ARGS, INITIALIZER_KWARGS
```
Comment (dizballanze) [inline tests/test_worker_pool.py:100]: Use closure instead of global variables

File: aiomisc/worker_pool/pool.py (hunk 1 above)
Comment (mosquito) [reply aiomisc/worker_pool/pool.py:204]: That fails the pool worker instance, it's more radical then just warning

File: tests/test_worker_pool.py (hunk 2 above)
Comment (mosquito) [reply tests/test_worker_pool.py:100]: It's will be returned from separate process, closure useless.

**Review by dizballanze — APPROVED** [review]

**Pull request merged**